ci/task_lint.sh

```diff
@@ -15,7 +15,7 @@ bash ./scripts/lint/git-black.sh HEAD~1
 bash ./scripts/lint/git-black.sh origin/main
 
 echo "Running pylint on slapo"
-python3 -m pylint slapo --rcfile=./scripts/lint/pylintrc
+python3 -m pylint -d duplicate-code slapo --rcfile=./scripts/lint/pylintrc
```

Comment:
Since now both Slapo and tests disable this rule, we could directly disable it in pylintrc so that we don't need to specify explicitly.